packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -43,7 +51,7 @@ export const Basic: StoryFn<typeof IconButton> = (args: IconButtonProps) => {
   return <IconButton {...args} />;
 };
 
-export const ExamplesSizes = () => {
+export const ExamplesSizes = (args: BasePropsWithTooltip) => {
```

Comment:
I think we should disable controls for these 'Examples' stories, it is a bit confusing that most of them don't do anything.

Reply:
I'll have a look at which props we can get rid of.

Reply:
Since controls are set for all stories at the same time I removed some props from the controls that don't make sense. I would like to to keep the others as they are used in the basic story.

Reply:
You can exclude controls from particular stories, look at `ConfirmModal` for example. You can do:

```
ExamplesSizes.parameters = {
  controls: {
    exclude: [],
  },
};
```

Reply:
Thanks a lot! I implemented it as you suggested.